docs/projects/template-fps/setup-and-installing.md

```diff
@@ -1 +1,27 @@
-// TODO: Populate this page from content from: https://github.com/improbable/ECS-FPS/blob/master/docs/setup-and-installing.md
+# Set up the FPS Starter Project
+
+1. Follow the [machine setup guide]({{urlRoot}}/setup-and-installing#set-up-your-machine).
+2. Clone the `GDK-for-Unity-FPS-Starter-Project` repository:
```

Comment:
How about adding a hyperlink to the fps starter project repo here?

Reply:
Yeah that's a good idea :)